tests/test_linear_incident_uptake_model.py

```diff
@@ -135,7 +143,9 @@ def test_build_scaffold_handles_groups():
     start_date = dt.date(2020, 2, 1)
     end_date = dt.date(2020, 2, 29)
     interval = "7d"
-    group_cols = ("geography",)
+    group_cols = [
+        "geography",
```

Comment:
```suggestion
        "geography"
```
ditto above

this is one of the weird things that I don't like, that the formatters will respect your comma as a statement that you don't want to collapse onto a single line

(I promise this is the last time I will point this out. The point of the formatter is so that we *don't* talk about style)

Reply:
No worries! I didn't know this, so it's helpful for you to point it out.